Disable cuda_distributions_test and converter_nomigraph_test on Windows.

ezyang opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#25305 Disable cuda_distributions_test on Windows.**

See https://github.com/pytorch/pytorch/issues/25304 and https://github.com/pytorch/pytorch/issues/25312

Signed-off-by: Edward Z. Yang <ezyang@fb.com>

Differential Revision: [D17092777](https://our.internmc.facebook.com/intern/diff/D17092777)

ezyang committed:
Disable cuda_distributions_test on Windows.

See https://github.com/pytorch/pytorch/issues/25304

Signed-off-by: Edward Z. Yang <ezyang@fb.com>

peterjc123 commented:
@ezyang We could use goto but it is still not so elegant.

peterjc123 commented:
The goto version could be like this:
```cmd
for /r "." %%a in (*.exe) do (
    call ::libtorch_check "%%~na" "%%~fa"
)

goto :eof

:libtorch_check
if "%~1" == "c10_metaprogramming_test" goto :eof
if "%~1" == "module_test" goto :eof
if "%~1" == "cuda_distributions_test" goto :eof

echo Running "%~2"
call "%~2"
if errorlevel 1 exit /b 1

goto :eof

```

ezyang committed:
Update on "Disable cuda_distributions_test on Windows."


See https://github.com/pytorch/pytorch/issues/25304

Signed-off-by: Edward Z. Yang <ezyang@fb.com>

Differential Revision: [D17092777](https://our.internmc.facebook.com/intern/diff/D17092777)

peterjc123 commented on .jenkins/pytorch/win-test-helpers/test_libtorch.bat:
```diff
@@ -6,17 +6,22 @@ test_api.exe --gtest_filter="-IntegrationTest.MNIST*"
 
 if errorlevel 1 exit /b 1
 
-cd %TMP_DIR_WIN%\build\torch\test
```
This line is necessary. Why is it removed?

peterjc123 replied:
Because the tests are skipped.
